cmd/generate/config/utils/generate.go

```diff
@@ -18,16 +18,17 @@ const (
 	identifierCaseInsensitivePrefix = `(?i:`
 	identifierCaseInsensitiveSuffix = `)`
 	identifierPrefix                = `(?:`
-	identifierSuffix                = `)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}`
+	identifierSuffix                = `)(?:[\w \t.-]{0,20})(?:\\*['"]{0,3})?[ \t]{0,5}`
```

Comment:
Side note: it can be handy to go ahead and escape quotes (e.g.  `[\'\"]`), even if it's not necessary in the regex here, because it makes it easier to copy/paste the regex into Splunk or other tools. But that's just a personal preference ^_^

Reply:
That seems reasonable to me. I'll add it.